maven-plugin/src/main/scala/org/apache/pekko/grpc/maven/AbstractGenerateMojo.scala

```diff
@@ -27,6 +27,7 @@ import protocbridge.{ JvmGenerator, ProtocRunner, Target }
 import scalapb.ScalaPbCodeGenerator
 
 import scala.beans.BeanProperty
+import scala.collection.JavaConverters._
```

Comment:
Were you meant to change this to `import org.apache.pekko.util.ccompat.JavaConverters._`?

Reply:
In this module, that class is not found. I guess I could look into changing the build for this module to add a dependency on the pekko-actor lib.

Reply:
I've repurposed this PR to only update the 'runtime' module